test/regression/expected/test_all_types.out

```diff
@@ -6,7 +6,6 @@ SELECT * exclude(
     tinyint, -- PG14 outputs this differently currently
     TIME,
     time_tz,
-    bit,
```

Comment:
The `docs/types.md` file should also be updated that these types are now supported.

Reply:
sure, add a new line for bit related types.